CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.14.6
+
+* Fix strong mode issue with ListEquality. The constructor of ListEquality
+  needs a default value which is constant, assignable to Equality<E> for
+  any E and which can compare elements of type E. This change makes const
+  DefaultEquality<Null>() a solution to that constraint.
```

Comment:
Nit: add backticks around type names and variables. This is especially important when writing generics, because otherwise the angle brackets will be interpreted as HTML tags and probably sanitized away.

```markdown
* Fix strong mode issue with `ListEquality`. The constructor of `ListEquality`
  needs a default value which is constant, assignable to `Equality<E>` for
  any `E` and which can compare elements of type `E`. This change makes
  `const DefaultEquality<Null>()` a solution to that constraint.
```